node/service/src/lib.rs

```diff
@@ -415,30 +444,140 @@ pub fn new_partial<RuntimeApi, Executor>(
 	config: &mut Configuration,
 	rpc_config: &RpcConfig,
 	dev_service: bool,
```

Comment:
If you keep the design where there are two separate functions, you no longer need this `dev_service` parameter in either one of them.